### ISSUE TITLE
Setting up MMdetction on Jetson Nano

### ISSUE THREAD

**Issue by SourabhRanade** [open-mmlab/mmdetection#7323]
I wanted to setup the mmdetection setup on a jetson nano. Has anyone managed to acheive the same?

**Comment by BIGWangYuDong** [comment]
https://github.com/open-mmlab/mmdetection/issues/1530 or https://github.com/open-mmlab/mmdetection/issues/4109 may help you.

**Comment by SourabhRanade** [comment]
@BIGWangYuDong  Up unitl now I have tried the following steps : 

Step 1: Installed nvidia precompiled wheel as pytorch downloaded from the official site fails to utilize to recognize the GPU in Jetson Nano. For this I have used the following link : [PyTorch for Jetson](https://forums.developer.nvidia.com/t/pytorch-for-jetson-version-1-10-now-available/72048)

Step 2 : Tried to follow the normal mmdetection installation from scratch using the following links : https://mmdetection.readthedocs.io/en/v2.2.0/install.html#a-from-scratch-setup-script

Step 2.1 : During compiling mmcv -full from source , the jetson nano simply hangs. MMCV lite installation goes through but if on running the following statement `import mmdet` I am thrown with the following error : `Error : No module named 'mmcv._ext'`. On refering the following issue : [#3271](https://github.com/open-mmlab/mmdetection/issues/3271), the suggestion is to install the mmcv -full. 

My Jetson Nano has been flashed and has nothing else installed in it. It has a 64GB SD card in it.
I am unable to proceed hereone.

**Comment by BIGWangYuDong** [comment]
Yes, seems that you need to install mmcv-full to avoid the error, using `MMCV_WITH_OPS=1 pip install -e .` to install mmcv-full from the source code. it will take a while to install the mmcv-full.
I haven't used Jetson Nano before. [Issue 1](https://github.com/open-mmlab/mmcv/issues/1720) and [issue 2](https://github.com/open-mmlab/mmcv/issues/1386)  in MMCV may help you.

**Comment by SourabhRanade** [comment]
@BIGWangYuDong Thank you ! With the initial steps that I mentioned and the link provided under Issue 1 , I was able to setup mmdetection on jetson nano.

**Comment by pinnintipraneethkumar** [comment]
hi, can you please explain in detail ?

**Comment by pinnintipraneethkumar** [comment]
I'm trying to install mmcv with this command pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu101/torch1.6.0/index.html it got stuck? will this command work on jetson nano?

**Comment by blackCmd** [comment]
I can't....
